docs/sources/whatsnew/whats-new-in-v8-2.md

```diff
@@ -1,42 +1,51 @@
-+++
 title = "What's new in Grafana v8.2"
 description = "Feature and improvement highlights for Grafana v8.2"
 keywords = ["grafana", "new", "documentation", "8.2", "release notes"]
 weight = -33
 aliases = ["/docs/grafana/latest/guides/whats-new-in-v8-2/"]
 [_build]
 list = false
+
 +++
 
 # What’s new in Grafana v8.2
 
-> **Note:** This topic will be updated frequently between now and the final release. Additionally, not all features listed here may be present in all the beta releases.
+Grafana 8.2 continues to build on the foundation of Grafana 8.0 & 8.1. Grafana 8.2 also marks the start of our work to bring Grafana closer to all users with a focus on increasing Grafana’s accessibility, part of its continuing mission to democratize metrics *for everyone*. 
 
-Grafana 8.2 continues to build on the foundation laid out in Grafana 8.0 & 8.1. We’ve continued to extend the GeoMap panel and bring new features to the Time Series panel. Grafana 8.2 also marks the start of our work to bring Grafana closer to all users with a focus on increasing Grafana’s accessibility, part of its continuing mission to democratize metrics _for everyone_.
+The plugin catalog is now on by default in Grafana 8.2. Using the plugin catalog you can now find and install official and community plugins without having to leave, or restart Grafana. We’ve also updated the time picker to include configurable fiscal quarters to make it easier to use Grafana to produce reports more closely aligned with common review and forecasting cycles. 
```

Comment:
```suggestion
The plugin catalog is now on by default in Grafana 8.2. Using the plugin catalog you can now find and install official and community plugins without having to leave or restart Grafana. We’ve also updated the time picker to include configurable fiscal quarters. This update makes it easier to use Grafana to produce reports more closely aligned with common review and forecasting cycles. 
```

Reply:
Just a suggestion:
The plugin catalog is enabled by default in Grafana 8.2. Use the plugin catalog to find and install official and community plugins without having to leave, or restart Grafana.